Source/SwiftLintFramework/Configuration+CommandLine.swift

```diff
@@ -241,6 +241,7 @@ extension Configuration {
             linters.asyncMap(visit)
     }
 
+    // swiftlint:disable:next no_async_without_await
```

Comment:
It seems that this method doesn't need to be `async`. Suppress or remove `async`?